src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3058,6 +3084,12 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
 
                 var ctor = addCtor(builder.RecordDeclarationWithParameters);
 
+                builder.UpdateIsNullableEnabledForConstructorsAndFields(ctor.IsStatic, compilation, paramList);
+                if (builder.RecordDeclarationWithParameters?.PrimaryConstructorBaseType?.ArgumentList is { } baseParamList)
```

Comment:
>? [](start = 59, length = 1)

The conditional nature of the access is unnecessary, we already checked that ```!(paramList is null)``` is true.